home/models.py

```diff
@@ -300,6 +300,36 @@ def clean(self, value):
         return result
 
 
+class SMSBlock(blocks.StructBlock):
+    message = blocks.TextBlock(
+        help_text="each message cannot exceed 160 characters.",
+        validators=(MaxLengthValidator(160),),
+    )
+
+    class Meta:
+        icon = "user"
+        form_classname = "whatsapp-message-block struct-block"
+
+
+class USSDBlock(blocks.StructBlock):
+    message = blocks.TextBlock(
+        help_text="each message cannot exceed 160 characters.",
+        validators=(MaxLengthValidator(160),),
+    )
+
+    class Meta:
+        icon = "user"
+        form_classname = "whatsapp-message-block struct-block"
+
+    def clean(self, value):
+        result = super().clean(value)
+        errors = {}
+
+        if errors:
+            raise StructBlockValidationError(errors)
+        return result
```

Comment:
I think we can probably do without this `clean` method? It doesn't seem to do anything from what I can see

Reply:
Yup, thx